README.md

```diff
@@ -69,7 +69,13 @@ there are two ways to specify the host in the jolokia URL so this URL will be re
 10. If you wish to run this as a service you need to create a service wrapper for it. Any pull requests for making it are welcome! If it's possible running it as docker making it simpler.
 
    
-
+## As Java Agent
+This lib can also get the metrics from MBean Platform instead of jolokia. In order to do so, we need to run inside the JVM.
```

Comment:
I think this should be described in the header as two different methods of getting JMX Metrics into Graphite - one through the running the docker image and second is installing a JVM Agent on the app it self - also specifying the use cases of each.